server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -776,13 +776,13 @@ private List<File> buildPods() throws IOException, InterruptedException, Extende
 
         List<String> paths = getFrameworkPaths(resolvedPods.frameworksDir);
         for (String path : paths) {
-            File[] frameworkDirs = new File(path).listFiles();
```

Comment:
This is nothing more than renaming of variables since a framework can be both a .framework package and a .a static library